bl31/aarch64/crash_reporting.S

```diff
@@ -350,6 +350,9 @@ func do_crash_reporting
 	/* Print the interconnect registers */
 	plat_print_interconnect_regs
 
+	/* Print some platform registers */
+	plat_print_misc_regs
+
```

Comment:
I can see the value of this but the new platform macro is a compatibility break but you've only updated some of the upstream platforms. Also the porting guide needs updating and a more meaningul commit message is needed.

Actually in hindsight we shouldn't have had 2 (now 3) platform macros for this when 1 will do. If we're going to introduce a break I'd prefer to resolve this at the same time. I think the new macro should be called `plat_crash_print_regs` and we should remove the calls to `plat_print_gic_regs` and `plat_print_interconnect_regs`. The implementations of `plat_crash_print_regs` should call local versions of the gic and interconnect print macros (or just move the code within `plat_crash_print_regs`).